include/fizzy/fizzy.h

```diff
@@ -133,6 +133,32 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Import definition.
+///
+/// @note Only one member of desc union corresponding to kind is defined for each import.
+typedef struct FizzyImport
```

Comment:
Since this is only used for inspection, perhaps I would call this something like `FizzyImportDetails`. The idea is to distinguish this from the regular API used for linking/execution.

Reply:
`FizzyImportType` perhaps?